generator/mavgen.py

```diff
@@ -168,11 +193,14 @@ def mavgen_python_dialect(dialect, wire_protocol):
     sys.stdout = io.StringIO()
     try:
         xml = os.path.relpath(xml)
-        mavgen( opts, [xml] )
+        if not mavgen( opts, [xml] ):
```

Comment:
May as well remove those extra spaces so when the style police come along its one less thing to fix :-)